### ISSUE TITLE
Clarification regarding future ext2/3/4 support?

### ISSUE THREAD

**Issue by dermotbradley** [Limine-Bootloader/Limine#251]
Hi

I see that 2 weeks ago a PR removed ext2/3/4 support and then shortly afterwards another PR reverted that.

So is the intention of ext2/3/4 support to stay? or is its reintroduction only a short-term "feature"?

As someone who creates Linux disk images (including with Limine) I'm trying to understand the long term situation...

**Comment by Mintsuki** [comment]
Hi

Let me start by mentioning that this change was *never* shipped with any existing Limine release. It was only done in `trunk`, and reverted in `trunk` without it ever hitting a stable branch.

The idea behind removing ext2/3/4 is to remove bloat from the bootloader that clashes with what's described in `PHILOSOPHY.md`.

That said, exceptions *can* be made if there is enough demand, and oh boy, there sure is :)

TL;DR ext2/3/4 is here to stay; and if it ever was dropped, the change wouldn't appear until a next major (compatibility breaking) release (aka not 4.x).
